.gitmodules

```diff
@@ -5,3 +5,6 @@
 [submodule "external/ebpf-for-windows"]
 	path = external/ebpf-for-windows
 	url = https://github.com/microsoft/ebpf-for-windows.git
+[submodule "external/Catch2"]
+	path = external/Catch2
+	url = https://github.com/catchorg/Catch2.git
```

Comment:
ebpf-for-windows already has catch2 as a submodule.  Can't you just use external\ebpf-for-windows\external\catch2 without adding it again here?

Reply:
Actually the commit of ebpf-for-windows repo that this repo is pointing to does not have catch2 as a submodule. I can fix this when I update the ebpf-for-windows submodule to latest.

Reply:
Filed Issue #8 to track this.